components/drawer/LeftDrawer.tsx

```diff
@@ -19,22 +18,21 @@ const LeftDrawer = observer(() => {
 
   return (
     <Drawer side="left">
-      <div className="w-full flex flex-col h-full bg-sky-100 dark:bg-black text-sky-600">
+      <div className="w-full flex flex-col h-full bg-fog-of-war dark:bg-black text-sky-600">
         <div
           className="flex mx-ztg-30 mt-ztg-20 h-ztg-35 mb-ztg-4"
           onClick={handleHomeClick}
           role="button"
         >
           <Logo />
           <div
-            className={`flex flex-col text-white  ml-ztg-14  ${
```

Comment:
Are you using the project prettier config? Seems like there are a few formatting changes

Reply:
That's not how it's being formatted on my side. If you click this in the bottom right:
<img width="126" alt="image" src="https://user-images.githubusercontent.com/4950844/203542089-c2a42c0c-7c20-4266-b236-f11dff22689b.png">
Do the prettier info logs match up with the `.prettierrc.js` file like this:
<img width="662" alt="image" src="https://user-images.githubusercontent.com/4950844/203542221-551ed319-ad52-4818-a319-daf2bdb185fe.png">

Reply:
fixed in [b7487cc](https://github.com/zeitgeistpm/ui/pull/441/commits/b7487cc5e58a2e52939b495f43fe110956a99fbe)